content/hegel/reference/existence/development.mdx

```diff
@@ -0,0 +1,221 @@
+task for the understanding is to distinguish the lack of determination in pure
+`being` from the determination in _a being_.
+
+The determination is not different from its being, Burbidge writes, as "the
+determination is rather the determinate way in which its being is present"
+(Burbidge 1981, 46). The determination also does not distinguish one being from
+another, since this would involve complexity that is not there in this
+indefinite sense of _a being_.
+
+### Houlgate
+
+Stephen Houlgate provides his reasoning as to why, in `existence`, `being` is
+united with `non-being`:
+
+> In _Dasein_ being is united with a nothing that is also different from it.
+> Insofar as nothing is different from being, however, it is itself inseparable
+> from being. It is thus no longer just nothing, but nothing or the “not”
+> (_Nicht_) that is explicitly connected to, and one with, being. The explicit
+> unity of the not and being is expressed in the thought of _not_-being or non-
+> being. Earlier in the Logic Hegel insists that nothing should initially be
+> thought as pure nothing, or the mere not, by itself, and so as “devoid of
+> relation” to being. Now, however, the not is inseparable from being: it is the
+> not, or “non”, as a form _of_ being. The negative moment with which being is
+> united in _Dasein_ is thus no longer nothing, but non-being (Houlgate 2022,
+> 158).
+
+Houlgate stresses the simplicity of `existence`: without the moment of
+`non-being`, being reverts to pure, _indeterminate_ `being` (_Sein_), but with
+that moment, however, `being` is converted to _determinate_ being or `existence`
+(_Dasein_). The moment of `non-being` constitutes determinacy as such. "Being is
+determinate, therefore, only because it is this-_not_-that" (Houlgate 2022,
+158).
+
+Why does `existence` emphasize the moment of `being`, or that `being` is what
+comes initially to the fore here? Houlgate answers that once `becoming` settles
+into `existence`, the former has vanished and the latter has emerged as the only
+remaining. There is no relation from one to the other. Houlgate writes, "it
+stands there alone as an '_immediate_', as simply itself. By virtue of that
+_immediacy_, however, it is 'in the one-sided determination of _being_': it
+simply _is_ what it _is_" (Houlgate 2022, 158). However, this one-sidedness is
+something we, as observers of the logical development, see. It is not something
+inherent in the immediacy of `existence` as such, "because _we_ do not see as
+much non-being in it as being" (Houlgate 2022, 158).
+
+While the inseparability of `being` with a negative seems uncontested, there
+could be issues with defining this negative element precisely (or too
+precisely). Houlgate uses "not", which seems to belong to the category of
+`negation`, and that is not derived at this stage in the _Logic_. In fact,
+`negation` becomes explicit in the section that follows this one, namely, the
+section on Quality.
```

Comment:
> "because we do not see as much non-being in it as being", we rather see it only as a form of being.

But this is what I struggle with. Do _we_ see it as a form of being or the _logic_? 

What does it mean to see something as much non-being as being? It's like that would require us to leap ahead into the category of `limit` but even there the `limit` also has being. 

Would it be right to say that, posited, there is only the immediacy of the unity of being and non-being -- but nothing more? The problem is injecting a determinateness at this stage where there really is none, or at least not made explicit?